apps/choleskey/CMakeLists.txt

```diff
@@ -21,3 +21,13 @@ target_include_directories(
 #   choleskey_stdpar_snd
 #   PRIVATE ${CMAKE_BINARY_DIR} ${CMAKE_CURRENT_LIST_DIR}/../../include
 #           ${ARGPARSE_INCLUDE_DIR} ${MDSPAN_INCLUDE_DIR})
+
+#tiled cholesky
+include_directories(/global/homes/c/chuanqiu/newrepo/locallib/openblasinstall/include/openblas)  
+link_directories(/global/homes/c/chuanqiu/newrepo/locallib/openblasinstall/lib64/cmake/OpenBLAS)
```

Comment:
please do not hardcode the path here since we have no access to your project. Consider add a cmake input variables like `STDPAR` or `OMP` variables but take the openblas directory.